src/components/DataInput/DataInput.cy.tsx

```diff
@@ -167,9 +167,13 @@ describe("Data Input Components", () => {
             cy.get("li[data-value='a']").click();
             cy.get("@onChangeSpy").should("have.been.calledWith", "a");
 
+            cy.get("div[class^='MuiSelect']").click();
```

Comment:
Why do we need this?

Reply:
This test breaks on actions about 1/8 times, adding a redundant get statement adds some more time and hopefully means reliability improves.

Reply:
Is there no other way to do this?

Reply:
There probably is, but I'm personally happy with this solution as it's concern is that the `Change handler for dropdown works`. The current issue is an asynchronously fired DOM change and not an issue with the dropdown. Feel free to experiment around with another solution, although it's tough to know what works when the failure happens so infrequently

Reply:
I also don't know if it's actually a solution, it's tough to tell but I haven't had any test failures on this specific test since I added this.

Reply:
Just experimented with the dropdown, and when you click a list item it closes. Not sure how the other test was passing but this is how the interaction would happen